packages/react-i18n/src/test/utilities.test.tsx

```diff
@@ -125,14 +125,23 @@ describe('translate()', () => {
       ]);
     });
 
-    it('uses the pluralization rules of the provided locale when a replacement named `count` is passed', () => {
-      const dictionary = {foo: {one: '{count} foo', other: '{count} foos'}};
-      expect(
-        translate('foo', {replacements: {count: 1}}, dictionary, locale),
-      ).toBe('1 foo');
-      expect(
-        translate('foo', {replacements: {count: 2}}, dictionary, locale),
-      ).toBe('2 foos');
+    describe('when a replacement named `count` is passed', () => {
+      it('uses the pluralization rules of the provided locale', () => {
+        const dictionary = {foo: {one: '{count} foo', other: '{count} foos'}};
+        expect(
+          translate('foo', {replacements: {count: 1}}, dictionary, locale),
+        ).toBe('1 foo');
+        expect(
+          translate('foo', {replacements: {count: 2}}, dictionary, locale),
+        ).toBe('2 foos');
+      });
+
+      it('formats the replacement `count`', () => {
+        const dictionary = {foo: {one: '{count} foo', other: '{count} foos'}};
+        expect(
+          translate('foo', {replacements: {count: 1000}}, dictionary, locale),
```

Comment:
It's weird if we pass `fr-fr` as locale, it still returns `1,000` 🤔 . 
It seems to only be in the tests though... 🤔

Reply:
After scratching my head on this one I think I found the reason from the [node.js docs](https://nodejs.org/api/intl.html#intl_build_with_a_pre_installed_icu_system_icu):

> Functionalities that only require the ICU library itself, such as String.prototype.normalize() and the WHATWG URL parser, are fully supported under system-icu. Features that require ICU locale data in addition, such as Intl.DateTimeFormat may be fully or partially supported, depending on the completeness of the ICU data installed on the system.

I ran the test they had on that page and our node.js build doesn't have the full ICU data, and to get it we would need to do a custom build. That explains why it always returns `1,000` without regards to the locale.

TL;DR our node.js build doesn't fully support Intl.

Reply:
Oh nice findings. 👍